src/components/claims/claim-details/components/ClaimType.tsx

```diff
@@ -68,241 +71,247 @@ const hasTicket = (typename: ClaimTypes): boolean => {
+      <SearchableDropdown
+        value={
+          type?.__typename &&
+          createClaimTypeOption(type?.__typename?.toString())
+        }
+        classNamePrefix="searchable-type-select"
+        placeholder={'What type of claim is this?'}
+        isLoading={setClaimTypeLoading || loadingClaimInformation}
+        isClearable={false}
+        isSearchable={true}
+        onChange={(selection) => {
+          setClaimType({
+            variables: { id: claimId, type: selection?.value ?? null },
+          })
+            .then(() => {
+              refetch()
             })
-            await refetchClaimInformation()
-            setSetClaimTypeLoading(false)
-          }}
-        >
-          {({ isValid }) => (
-            <Form>
-              <div>
-                <Field component={FieldSelect} name="selectedType">
-                  <MuiMenuItem disabled value="">
-                    Select a type...
-                  </MuiMenuItem>
-                  {Object.keys(ClaimTypes).map((t) => (
-                    <MuiMenuItem key={t} value={t}>
-                      {t}
-                    </MuiMenuItem>
-                  ))}
-                </Field>
-              </div>
-              <div>
-                <SubmitButton
-                  type="submit"
-                  variant="contained"
-                  color="primary"
-                  disabled={
-                    !isValid ||
-                    setClaimTypeProps.loading ||
-                    isSetClaimTypeLoading
-                  }
-                >
-                  Set type
-                  {(setClaimTypeProps.loading || isSetClaimTypeLoading) && (
-                    <Spinner push="left" />
-                  )}
-                </SubmitButton>
-              </div>
-            </Form>
-          )}
-        </Formik>
-        {type ? (
-          <Formik<{
-            location?: string
-            date: Date | null
-            item?: string
-            policeReport?: string
-            receipt?: string
-            ticket?: string
-          }>
-            initialValues={{
-              location: typeAny?.location || '',
-              date: typeAny?.date ? parseISO(typeAny.date) : null,
-              item: typeAny?.item || '',
-              policeReport: typeAny?.policeReport || '',
-              receipt: typeAny?.receipt || '',
-              ticket: typeAny?.ticket || '',
-            }}
-            onSubmit={async (values) => {
-              await setClaimInformation({
-                variables: {
-                  id: claimId,
-                  claimInformation: {
-                    ...values,
-                    date: values.date && format(values.date, 'yyyy-MM-dd'),
-                  },
-                },
+            .catch(() => {
+              showNotification({
+                header: 'Error',
+                type: 'red',
+                message: 'Something went wrong, notify the tech team',
```

Comment:
Maybe we can print the message instead
```suggestion
                message: e.message,
```